context("test")

test_that("Input by user works",{
  expect_error()
})
